SitumWayfinding/Classes/UI/UserInterfaceTheme/Extensions/UIViewExtension.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  UIImageExtension.swift
+//  SitumWayfinding
+//
+//  Created by fsvilas on 17/1/23.
+//
+
+import Foundation
+
+extension UIView{
+    func setSitumShadow(enabled:Bool){
```

Comment:
Polo que esto sempre se chama cunha propiedade do obxecto UIColorsTheme, creo que quedaría máis explícto e encapsulado pasarlle este obxecto e usar esa propiedade no if para saber se aplicar a shadow ou non. Así se maña esta configuración requirere mirar algo máis concreto xa estamos cubertos a posibles cambios sen cambiar interfaces

Un espacio entre enabled: e Bool e eoutro entre Bool) e {